partner_address_version/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module allows to manage versions for partner addresses.
```

Comment:
Can you write a sentence about the behavior when writting on a field that is versionned?

And maybe also why versionning is useful, for instance for e-commerce to track history.